dist/common/scripts/scylla_coredump_setup

```diff
@@ -99,6 +99,7 @@ WantedBy=multi-user.target
         try:
             run('coredumpctl --no-pager --no-legend info {}'.format(pid))
             print('\nsystemd-coredump is working finely.')
+            run('rm /var/lib/systemd/coredump/core.bash.*')
```

Comment:
better `rm -f `

Reply:
> I think it is better to locate the file to delete the way cluster.py at sdcm does it.
> see _notify_backtrace function.

It's not just a problem in testing env, we should not leave a meaningless core file to users, it's not graceful.

Reply:
@fruch do you agree to use '-f' option?

Reply:
I agree